.github/workflows/docs.yml

```
@@ -0,0 +1,125 @@
+on:
+  pull_request:
+  push:
+    branches: [master]
+
+name: Docs
+
+jobs:
+  set-image:
+    runs-on: ubuntu-latest
+    outputs:
+      IMAGE: ${{ steps.set_image.outputs.IMAGE }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - id: set_image
+        run: cat .github/env >> $GITHUB_OUTPUT
+  test-rustdoc:
+    runs-on: arc-runners-polkadot-sdk-beefy
+    needs: [set-image]
+    container:
+      image: ${{ needs.set-image.outputs.IMAGE }}
+    steps:
+      - uses: actions/checkout@v4
+      - run: time cargo doc --workspace --all-features --no-deps
```

```suggestion
      - run: forklift cargo doc --workspace --all-features --no-deps
```
There is no way to set alias (`alias cargo="forklift cargo"`) for all workflows as we do in gitlab so we have to specify it explicitly.